src/libs/antares/study/parameters.cpp

```diff
@@ -74,6 +74,8 @@ static bool ConvertCStrToListTimeSeries(const String& value, uint& v)
             v |= timeSeriesRenewable;
         else if (word == "ntc")
             v |= timeSeriesTransmissionCapacities;
+        else if (word == "power-credits")
```

Comment:
As said somewhere else, **power** and **credit** not compatible : **hydro-max-power** is better;
Same for `timeSeriesHydroPowerCredits`

Reply:
Done!